opentok/opentok.py

```diff
@@ -1314,6 +1318,8 @@ def start_broadcast(self, session_id, options, stream_mode=BroadcastStreamModes.
             String 'resolution' optional: The resolution of the broadcast, either "640x480"
             (SD, the default) or "1280x720" (HD)
 
+            String 'multiBroadcastTag' optional: If specified, triggers a new simultaneous broadcast on the session.
```

Comment:
As a suggestion this should be a camel case param just like `multi_archive_tag ` above

Reply:
Did you mean snake_case? Will change for consistency

Reply:
As this option is passed through directly, as are the others in the `options` dict, the camel case makes more sense here imo